src/plugins/util/web.js

```diff
@@ -92,8 +92,6 @@ const web = {
       [RESOURCE_NAME]: middleware._name || middleware.name || '<anonymous>'
     })
 
-    analyticsSampler.sample(span, req._datadog.config.analytics)
```

Comment:
Why are we not sampling this anymore?

Reply:
Without the per span name configuration this would never be sampled, so it was simply removed.